staging/src/k8s.io/code-generator/cmd/conversion-gen/main.go

```diff
@@ -106,6 +106,9 @@ func main() {
 	// TODO: move this out of conversion-gen
 	genericArgs.GoHeaderFilePath = filepath.Join(args.DefaultSourceTree(), util.BoilerplatePath())
 
+	// Import generated source code for finding conversion functions by default
+	genericArgs.ImportBuildTag = ""
```

Comment:
what does this do?

Reply:
I explained it here: https://github.com/kubernetes/kubernetes/issues/101567